libpivot/include/pivot/game_engine.hxx

```diff
@@ -0,0 +1 @@
+#pragma once
```

Comment:
Why an empty file ?

Reply:
Placeholders for the game engine code when I set up the repo. I will remove them